code/game/objects/items/devices/flash.dm

```diff
@@ -66,12 +66,14 @@
 			M.confused += power
 			terrible_conversion_proc(M, user)
 			M.Stun(1)
-			user.visible_message("<span class='disarm'>[user] blinds [M] with the flash!</span>")
+			user.visible_message("<span class='disarm'>[user] blinds [M] with the flash!</span>", "<span class='danger'>You blind [M] with the flash!</span>")
+			M << "<span class='userdanger'>[user] blinds you with the flash!</span>"

```

Comment:
Wait I just realized... Does M get two messages now? Like:
"[User] blinds [M]!"
"[User] blinds you!"?

I don't know how to test this on my own server.